aspnetcore/migration/22-to-30.md

```diff
@@ -1048,18 +1048,13 @@ For more information, see [Breaking change in `AddAuthorization(o =>`) overload
 
 ## Identity UI
 
-Complete the following steps:
+Identity UI updates for ASP.NET Core 3.0
 
 * Add a package reference to [Microsoft.AspNetCore.Identity.UI](https://nuget.org/packages/Microsoft.AspNetCore.Identity.UI).
-* If the app isn't already using Razor Pages, add a call to `MapRazorPages`. See [Razor Pages](#razor-pages) in this document.
-* Prior to ASP.NET Core 3.0, the default UI framework was Bootstrap 3. The UI framework could be set to Bootstrap 4 with the following code:
+* Apps that don't use Razor Pages must call `MapRazorPages`. See [Razor Pages](#razor-pages) in this document.
```

Comment:
This is much better. I didn't really like the if, but couldn't think of anything better.